crates/store/src/db/sql.rs

```diff
@@ -490,7 +488,6 @@ pub fn select_notes_since_block_by_tag_and_sender(
         let note = Note {
             block_num,
             note_created: NoteCreated {
```

Comment:
It is a bit confusing to me why we have these `Note` and `NoteCreated` structs. We'll probably need to revisit this in the future.

Reply:
Ah, you're right, it's useless now, so I merged them into one struct `Note`.